eclcli/vnf/v1/virtual_network_appliance.py

```diff
@@ -121,7 +122,7 @@ def get_parser(self, prog_name):
                    "virtual network appliance. "
                    "net-id: attach interface to network with this UUID, "
                    "ip-address: IPv4 fixed address for interface. "
-                   "(You can specify only one address in creation), "
+                   "(You can specify addresses in creation), "
```

Comment:
```suggestion
                   "(You can specify one or more addresses in creation), "
```
It would be better to be more specific for the number of addresses that users can specify.